packs/spell-effects/spell-effect-dragon-form.json

```
@@ -0,0 +1,426 @@
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "fire",
+                            "dragonType": "diabolic",
+                            "saveType": "reflex",
+                            "tradition": "divine"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Dragon.Empyreal",
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "spirit",
+                            "dragonType": "empyreal",
+                            "saveType": "reflex",
+                            "tradition": "divine"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Dragon.Fortune",
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "force",
+                            "dragonType": "fortune",
+                            "saveType": "reflex",
+                            "tradition": "arcane"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Dragon.Horned",
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "poison",
+                            "dragonType": "horned",
+                            "saveType": "fortitude",
+                            "tradition": "primal"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Dragon.Mirage",
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "mental",
+                            "dragonType": "mirage",
+                            "saveType": "will",
+                            "tradition": "arcane"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Dragon.Omen",
+                        "value": {
+                            "breathShape": "cone",
+                            "damageType": "mental",
+                            "dragonType": "omen",
+                            "saveType": "will",
+                            "tradition": "occult"
+                        }
+                    }
+                ],
+                "flag": "dragonForm",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.DragonDisciple.Prompt"
+            },
+            {
+                "key": "RollOption",
+                "option": "dragon-form:{item|flags.pf2e.rulesSelections.dragonForm.dragonType}"
+            },
+            {
+                "key": "RollOption",
+                "option": "dragon-tradition:{item|flags.pf2e.rulesSelections.dragonForm.tradition}"
```

```suggestion
                "option": "dragon-form-tradition:{item|flags.pf2e.rulesSelections.dragonForm.tradition}"
```
Best to disambiguate this a bit more